2-farm/lessons/1-predict-plant-growth/README.md

```diff
@@ -265,6 +265,7 @@ For these, find if there are sensors that can measure them. What about actuators
 * The full growing degree days calculation is more complicated than the simplified one given here. Read more about the more complicated equation and how to deal with temperatures below the baseline on the [Growing Degree Day Wikipedia page](https://wikipedia.org/wiki/Growing_degree-day).
 * Food may be scarce in future even we still use same methods for farming. Learn more about hi-tech farming techniques in this [Hi-Tech Farms of Future video on YouTube](https://www.youtube.com/watch?v=KIEOuKD9KX8).
 
+Food may be scarce in future even we still use same methods for farming. Here you can see hi-tech farming techniques. [Hi-Tech Farms of Future](https://www.youtube.com/watch?v=KIEOuKD9KX8)
```

Comment:
Can you remove this please - this was added to the review and self study section

Reply:
@jimbobbennett  I've done that in #241